internal/k8s/crdcontroller.go

```diff
@@ -315,8 +315,23 @@ func checkRefOnController(key, refKey, refValue string) error {
 	}
 
 	if result.Count == 0 {
-		utils.AviLog.Warnf("key: %s, msg: No Objects found for refName: %s/%s", key, refModelMap[refKey], refValue)
-		return fmt.Errorf("%s \"%s\" not found on controller", refModelMap[refKey], refValue)
+		if refModelMap[refKey] == "network" {
+			var rest_response interface{}
+			uri = fmt.Sprintf("/api/%s/%s", refModelMap[refKey], refValue)
```

Comment:
can we determine preemptively whether the user would provide uuids or names by checking the cloudType?

Reply:
i believe user will always provide subnet uuid in case of AWS cloud, since we do not accept subnet name on the controller for AWS.